moderatorguide.md

```diff
@@ -36,7 +38,7 @@ Unless otherwise agreed by the organizers, the following guidelines apply:
 * Short talks: after the talk ends.
 * Long talks: questions may be asked at any time.
 
-In general, mediating questions works like this:
+Mediating questions:
 
 * Monitor the participants list to observe attendees signaling for questions (“Raise hand option”).
 * Let speaker and audience know that there is a question, and signal the asker by stating their name and unmuting their microphone.
```

Comment:
Perhaps add a line mentioning that the moderator decides when to stop the speaker for the question?